README.md

```diff
@@ -171,14 +172,15 @@ _This library is built for who uses GraphQL on NodeJS, you can use model to oper
 
   - Props: `object`
 
-    | Key           | Type     | Introduction                       | Default value |
-    | ------------- | -------- | ---------------------------------- | ------------- |
-    | host          | `string` | Postgresql server host             | "localhost"   |
-    | port          | `number` | Postgresql server port             | 5432          |
-    | userName      | `string` | Postgresql server user name        | "postgres"    |
-    | password      | `string` | Postgresql server password         | ""_(empty)_   |
-    | database      | `string` | Postgresql database name           | "postgres"    |
-    | connectionMax | `number` | Postgresql database max connection | 10            |
+    | Key                     | Type            | Introduction                                                 | Default value |
```

Comment:
table seems to got all messed up agaib

Reply:
I think no bad just something too long. Could you pull it to you local and try it?